### PR TITLE
javascript file for missing functions on ie8

### PR DESCRIPTION
**Pull request by namiltd**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.

### CODE REVIEW

**Comment by namiltd** [comment]
'''
    
    
    
'''

**Comment by namiltd** [comment]
It repairs the opening of modal dialogs (bootstrap-modal and bootstrap-util modules) on ie8.

**Comment by namiltd** [comment]
See test.htm on my repository: https://github.com/namiltd/bootstrap-ie8

**Comment by coliff** [comment]
Great work on this so far. I've tested in IE8 and modals are working well - are you working on fixing any of the other JS components at all?

Do you know why this does this not work with Bootstrap 4.1 release version? Do we need to wait for 4.1.1 or 4.2 to drop officially before folks can use this with a CDN version of Bootstrap 4.x ?

Thanks again for the PR and your work so far. 👍

**Comment by namiltd** [comment]
Works correctly after my fix https://github.com/twbs/bootstrap/commit/418fd76ba97358730e4ae7fd28f73dbb5e275fbd

File: css/bootstrap-ie8.css [hunk 1]
```diff
@@ -76,8 +76,9 @@ button.close, .btn{cursor:pointer}
 .carousel-indicators li{background-color:#fff}
 .checkbox input[type=checkbox],.checkbox-inline input[type=checkbox],.radio input[type=radio],.radio-inline input[type=radio]{margin-top:4px}
 .clearfix:after{content:"";display:table;clear:both}
-.close{font-size:24px;filter:alpha(opacity=50)}
+.close{cursor:pointer;font-size:24px;filter:alpha(opacity=50)}
 .close:focus,.close:hover{filter:alpha(opacity=75)}
+.close:disabled{cursor:auto}
```
Comment (coliff) [inline css/bootstrap-ie8.css]: The `:disabled` selector isn't supported on IE8 so don't need to add this line.

**Comment by namiltd** [comment]
Updated test on: http://namiltd.github.io/bootstrap-ie8/test.htm

**Pull request merged**